### PR TITLE
Correctly handle base paths SPA routing and deeplinks

### PR DESCRIPTION
**Pull request by marlonbaeten**
See #59

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,33 +4,33 @@
   "version": "0.4.0",
   "type": "module",
   "scripts": {
-    "dev": "ROTO_API_HOST=https://rest.bgp-api.net ROUTINATOR_API_HOST=https://routinator.do.nlnetlabs.nl vite",
+    "dev": "ROTO_API_HOST=https://rest.bgp-api.net vite",
```
Comment (partim) [inline package.json:7]: Shouldn’t dev still have have `ROUTINATOR_API_HOST=https://routinator.nlnetlabs.nl`?

Comment (marlonbaeten) [reply package.json:7]: The problem is that CORS headers will prevent the API from being loaded in development. I've added a server proxy configuration te circumvent this when running `yarn dev`.

Comment (partim) [reply package.json:7]: I’m trying to remember: Who would need to set the CORS headers to allow the link? routinator.nlnetlabs.nl, right?

Comment (marlonbaeten) [reply package.json:7]: The CORS header (`Access-Control-Allow-Origin`) would be necessary on the API endpoint to allow browsers to load it from any other origin than routinator.nlnetlabs.nl.

Comment (partim) [reply package.json:7]: Hm, actually, the server proxy configuration is quite neat, so let’s just stick to that.

**Review by partim — APPROVED** [review]

**Comment by partim** [comment]
Thank you for the quick PR! I shall release 0.4.1 now.

**Pull request merged**